### ISSUE TITLE
An option to resize Nemo left panel icons/buttons

### ISSUE THREAD

**Issue by Sven71** [linuxmint/nemo#2607]
Feature request: it would be nice to have an option to change the size of Nemo left panel icons/buttons. You know, setting larger targets to click on for older people with worse hand-eye coordination.

**Comment by linuxisfreedom** [comment]
Yes, this is one of the reasons why I use Dolphin. There, you can easily switch the size of the sidebar symbols.
